### PR TITLE
Follow Up PR from Color enums

### PR DESCRIPTION
**Pull request by kolibril13**
Follow up for https://github.com/ManimCommunity/manim/pull/488

### CODE REVIEW

File: manim/utils/color.py [hunk 1]
```diff
@@ -115,8 +115,9 @@ class Colors(Enum):
 
     .. code-block:: python
 
-       from manim.utils.color import Colors
-       Colors.white.value   # -> '#FFFFFF'
+        >>> import manim.utils.color as C
+        >>> C.WHITE
+        '#FFFFFF'
```
Comment (behackl) [inline manim/utils/color.py:120]: Both code-blocks in this file should be doctests, and both should do the same thing as before:

```suggestion
        >>> from manim.utils.color import Colors
        >>> Colors.white.value
        '#FFFFFF'
```

**Review by behackl — CHANGES_REQUESTED** [review]

**Review by leotrs — APPROVED** [review]

**Review by leotrs — CHANGES_REQUESTED** [review]
GH won't allow me to leave a coment on L 107, but you forgot to put the `>>>`. Please give it the same treatment that @behackl asked for the other code block.

**Review by behackl — APPROVED** [review]
This looks good to me. With the doctests taking over the test from `tests/test_color.py`, we could remove the file again. I might have a slight preference for removing it to keep only actually necessary files around, but would also be okay with keeping it around.

File: manim/utils/color.py [hunk 2]
```diff
@@ -104,8 +104,9 @@ class Colors(Enum):
 
     .. code-block:: python
 
-       import manim.utils.color as C
-       C.WHITE   # -> '#FFFFFF'
+        >>> import manim.utils.color as C
+        >>> C.WHITE
```
Comment (behackl) [inline manim/utils/color.py:108]: Sorry, just a tiny issue: the trailing whitespace should be removed.

```suggestion
        >>> C.WHITE
```

**Review by behackl — CHANGES_REQUESTED** [review]

File: manim/utils/color.py (hunk 2 above)
Comment (kolibril13) [reply manim/utils/color.py:108]: ok :D

**Review by behackl — APPROVED** [review]

**Review by leotrs — APPROVED** [review]

**Pull request merged**